src/helper.js

```diff
@@ -76,6 +78,27 @@ export default class Helper {
       }));
     }
 
+    static addCheckboxEvent = () => {
+      document.querySelectorAll("input[type='checkbox']").forEach((box) => box.addEventListener('change', () => {
+        const id = box.id - 1;
+        let currentStatus;
+        if (box.checked === true) {
+          currentStatus = true;
+        } else if (box.checked !== true) {
+          currentStatus = false;
+        }
+
+        this.toggleCompleted(id, currentStatus);
+      }));
+    }
+
+    static toggleCompleted = (id, currentStatus) => {
+      const taskList = Helper.getLocalStorageList();
+      taskList[id].completed = currentStatus;
+      Helper.setLocalStorageList(taskList);
+      Helper.showTaskItems();
+    }
```

Comment:
This part of my code, updates the array of changes in task status and stores in local storage @andersonlebon .